src/classes/issues-processor.ts

```diff
@@ -268,6 +315,109 @@ export class IssuesProcessor {
       );
     }
 
+    if (this.options.mlflow) {
+      if (issue.isPullRequest) {
+        // TODO
+      } else {
+        const hasClosingPr = issue.labels.some(
+          ({name}) => name === 'has-closing-pr'
+        );
+        if (hasClosingPr) {
+          issueLogger.info('This issue has a closing PR');
+          return;
+        }
```

Comment:
I propose we add a github action workflow to apply a `has-closing-pr` label to an issue.

Reply:
Without the label, Can we directly check whether an issue has a related PR with "closing" ?

Reply:
We can't. There is no GitHub API that can get linked issues/PRs.

Reply:
https://docs.github.com/en/rest/issues/events might be able to fetch linked issues.

Reply:
[Issue Events API](https://docs.github.com/en/developers/webhooks-and-events/events/issue-event-types) might be able to fetch linked issues/PRs, but requires another API call.